### ISSUE TITLE
Braille: Cannot exit input help mode with emulated NVDA key.

### ISSUE THREAD

**Issue by derekriemer** [nvaccess/nvda#9633]
### Steps to reproduce:

1. connect braille display, and set up a key on the display to emulate the NVDA key with.
2. Press the NVDA key. NVDA says "NVDA pressed"
3. Press 1. Input help turns on.
4. Emulate NVDA again. it says "NVDA pressed.
5. press 1.

### Actual behavior:
it says 1.

### Expected behavior:
Input help turns off.

**Comment by derekriemer** [comment]
@leonardder

**Comment by leonardder** [comment]
Ugh, this is a tricky one. Definitely not high on my priority list at 
the moment. Feel free to assign this a priority yourself, I guess p3 
will suffice.

**Comment by Brian1Gaff** [comment]
Is it just the one toggle or does this affect others as well?
 Brian

bglists@blueyonder.co.uk 
Sent via blueyonder. 
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
This message sent from a Windows XP machine!

**Comment by Adriani90** [comment]
cc: @burmancomp, @Emil-18 maybe you have some suggestion on how to solve this.
